engine-plugins/connect-plugin/pom.xml

```diff
@@ -33,10 +33,16 @@
     <dependency>
       <groupId>org.operaton.bpm</groupId>
       <artifactId>operaton-bpm-junit5</artifactId>
-      <version>${project.version}</version>
       <scope>test</scope>
     </dependency>
```

Comment:
These dependencies can be removed now

Reply:
Removed dependencies.

Reply:
Added junit-vontage to ensure Junit 4 Tests are also executed on operaton-engine.